### ISSUE TITLE
[a11y]: Slider - Controlled Slider - JAWS is not announcing the Randomize value

### ISSUE THREAD

**Issue by 2nikhiltom** [carbon-design-system/carbon#16634]
### Package

@carbon/react

### Browser

Safari

### Package version

v1.58.0

### React version

18.02

### Automated testing tool and ruleset

Manual QA

### Assistive technology

JAWS

### Description

When the user activates the Randomize Value button, the slider value changes, but JAWS does not announce the new value.

### Reproduction/example

https://react.carbondesignsystem.com/?path=/story/components-slider--controlled-slider

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by 2nikhiltom** [comment]
When the `focus` is on the `button` and the slider value changes via an onClick function, the screen reader won't announce the update since it remains focused on the button and not the Slider component . 
Using a live region to announce the change could cause double-speaking issues when the user interacts directly with the slider. Alternatively, shifting focus to the slider could trigger the announcement but may disrupt the user’s workflow.
